### PR TITLE
externalize strings

### LINKED ISSUE #494: Move external messages into message bundle in com.google.cloud.tools.eclipse.appengine.newproject

### PR DESCRIPTION
**Pull request by elharo**
@nbashirbello fix #494

### CODE REVIEW

**Comment by chanseokoh** [comment]
It'd be nice to add `//$NON-NLS-1$` to all externalized strings, but not a critical problem.

**Comment by elharo** [comment]
I was hoping that Eclipse was smart enough to realize that a string literal in Messages.getString() did not itself need to be localized, but I'm not actually sure that's true.

**Comment by elharo** [comment]
done PTAL @chanseokoh

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties [hunk 1]
```diff
@@ -0,0 +1,12 @@
+new.app.engine.standard.project=New App Engine Standard Project
+creating.app.engine.standard.project=Creating App Engine Standard Project
+adding.app.engine.libraries=Adding App Engine Libraries
+creating.new.app.engine.standard.project=Creating new App Engine Standard Project
+app.engine.standard.project=App Engine Standard Project
+create.app.engine.standard.project=Create a new App Engine Standard Project in the workspace.
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties]: I noticed you changed the corresponding message to be Standard _Environment_ Project in #1056 for the Maven-based wizard.

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java [hunk 2]
```diff
@@ -103,19 +107,21 @@ private void addAppEngineLibrariesToBuildPath(IProject newProject,
     if (libraries.isEmpty()) {
       return;
     }
-    SubMonitor subMonitor = SubMonitor.convert(monitor, "Adding App Engine libraries", libraries.size());
+    SubMonitor subMonitor = SubMonitor.convert(monitor,
+        Messages.getString("adding.appengine.libraries"), libraries.size()); //$NON-NLS-1$
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java]: Either this should be `adding.app.engine.libraries` (dot between `app` and `engine`), or that dot should be removed from `messages.properties`.

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java [hunk 3]
```diff
@@ -146,14 +153,16 @@ public void done(IJobChangeEvent event) {
     job.schedule();
   }
 
-  private void addJunit4ToClasspath(IProgressMonitor monitor, final IProject newProject) throws CoreException,
-                                                                                         JavaModelException {
+  private static void addJunit4ToClasspath(IProgressMonitor monitor, final IProject newProject)
+      throws CoreException, JavaModelException {
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java]: Not related, but seems like we can remove `JavaModelException`.

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+project.creation.failed=Failed to create project
+java.package=Java package:
+illegal.package.name=Illegal package name: {0}
+project.location.exists: Project location already exists: {0}
+package.ends.with.period: {0} ends with a period.
+package.contains.whitespace: "{0}" contains whitespace
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties]: Period at the end, as the line above.

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java (hunk 2 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java (hunk 3 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties (hunk 1 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties (hunk 4 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/messages.properties]: done

**Comment by elharo** [comment]
ping

**Comment by chanseokoh** [comment]
Did you push your commit?

**Comment by elharo** [comment]
$ git push origin i494
Enter passphrase for key '/Users/elharo/.ssh/id_rsa': 
Everything up-to-date

**Comment by elharo** [comment]
But I forgot to commit. Oops. Done.

**Pull request merged**